pages/index.tsx

```diff
@@ -48,8 +48,10 @@ export default function Home() {
 			.then((response) => response.json())
 			.then((data) => {
 				let projectHtml = [];
-				if (data.length <= 3) {
-					projectHtml = data.map((project: IProject) => (
+
+				projectHtml = data
+					.filter((project: IProject) => project.status === 'ongoing')
+					.map((project: IProject) => (
```

Comment:
Put a max limit on this please, otherwise this is just another projects page.

Reply:
All right I put it back to 3, it's basically the same as before but now filtering out past projects.